.github/workflows/main.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: "3.8"
```

Comment:
Why not Python3.8 here?

Reply:
woups I haven't pushed my other revert commit. We should use python 3.8 indeed.